### PR TITLE
Add a WebSocket component

### LINKED ISSUE #6: Add websocket component
This will use [socket.io](https://socket.io/).

### PR DESCRIPTION
**Pull request by j-f1**
Closes #6.

TODO:
- [x] Install socket.io
- [x] Create ws.js to manage websockets
- [x] Set `ws: true` on routes to make them channels for websockets
- [x] Add tests
- [x] Add docs

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -111,7 +97,9 @@ will have access to any data you pass in the `locals` option to the `render` cal
 ## 4. Test it!
 You now have a complete nails app. To test it, run your startup script using Node:
 
-    $ node startup.js
+```
+$ node startup.js
```
Comment (ArtOfCode-) [inline docs/getting-started.md]: s/startup/server/

Comment (j-f1) [inline docs/getting-started.md]: Fixed. Thanks!

File: docs/api/ws.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ws('/status', 'status')
+  ```
+4. Include this `<script>` tag in your HTML page:
+  ```html
+  <script src="/socket.io/socket.io.js" charset="utf-8"></script>
+  ```
+5. Create the socket and join the appropriate room:
+  ```js
+  const socket = io();
+  window.sock.emit('join', '/status')
```
Comment (ArtOfCode-) [inline docs/api/ws.md]: Should this be `window.sock` or `socket`?

Comment (j-f1) [inline docs/api/ws.md]: That’s what happens when I copy-paste. Fixed!

**Pull request merged**